Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2875,12 +2875,12 @@ FileReaderAPIEnabled:
       default: true
   disableInLockdownMode: true
 
-FileSystemAccessEnabled:
```

Comment:
If a user changed this setting in, e.g. Safari, won't this rename wipe out that preference when they upgrade? Can we live with the preference being named `FileSystemAccessEnabled` but change the human readable name and description?

Reply:
I think you want `status: mature` in that case.

Reply:
AFAIK it's not required to be on, it just defaults to on. I don't maintain this API though so I don't know if that's a good idea. This just caught my eye since the rename would cause any user who chose to turn it off (by jumping through the multiple hoops to do so) would find it back on when upgrading their browser.